include/mbedtls/platform_util.h

```diff
@@ -167,7 +167,17 @@ MBEDTLS_DEPRECATED typedef int mbedtls_deprecated_numeric_constant_t;
  * \param len   Length of the buffer in bytes
  *
  */
+#if !defined(MBEDTLS_TEST_DEFINES_ZEROIZE)
 void mbedtls_platform_zeroize(void *buf, size_t len);
+#endif
+
+/* MBEDTLS_TEST_DEFINES_ZEROIZE
+ *
+ * Indicates that the library is being built by the test framework, and the
+ * framework is going to provide a replacement mbedtls_platform_zeroize()
+ * using a pre-processor macro, so the function declaration should be omitted.
+ */
+//#define MBEDTLS_TEST_DEFINES_ZEROIZE
```

Comment:
> This line is pointless

Without this line, check-names complains.

> defining the macro at this point isn't going to have the desired effect

One of the reasons for defining it after it was checked was so that it couldn't be accidentally enabled and thereby cause zeroize calls to be removed

Reply:
> Without this line, check-names complains.

Oh, well, that's a job for `//no-check-names`, which I see you've found.